### PR TITLE
Practical Support Config

### LINKED ISSUE #1619: Add practical support config option
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**

We'll want funds to be able to set their own categories of practical support. We should do this with a config so someone can just add this in the admin panel and we aren't getting fancy.

* **What feature or behavior is this required for?**

Practical support implementation

* **How could we solve this issue? (Not knowing is okay!)**

* new config for practical support options
* should just be comma separated strings
* should have a practical_support_options helper method that works the same way the insurance one does

* **Anything else?**

stems from #1493

### PR DESCRIPTION
**Pull request by lomky**
I rule and have completed some work on Case Manager that's ready for review!

Implementation & tests for the Practical Support Config options!

This pull request makes the following changes:
* Adds the practical support config
* Adds a helper for displaying them
* Includes i18n support
* Helper tests
* Config system test

Not in this ticket:
* Any tests relying on the patient view of practical support (probably should do in/after #1602)

screenshot from 2019-02-24 08-46-52

It relates to the following issue #s: 
* Fixes #1619
* Bump #1493

### CODE REVIEW

**Comment by lomky** [comment]
Ready for Spanish review! @montanezp 

Is there a good option for a gender neutral 'Companion' in Spanish?

File: config/locales/en.yml [hunk 1]
```diff
@@ -177,6 +177,11 @@ en:
         unknown: Don't know
       language:
         English: English
+      practical_support:
+        companion: Companion
```
Comment (colinxfleming) [inline config/locales/en.yml]: @harumhelmy can you verify that these are good as starter, general categories of practical support?

File: test/test_helper.rb [hunk 2]
```diff
@@ -45,6 +45,12 @@ def create_insurance_config
                     config_value: { options: insurance_options }
   end
 
+  def create_practical_support_config
+    practical_support_options = ['Metallica Tickets', 'Clothing']
```
Comment (colinxfleming) [inline test/test_helper.rb]: 🤘

File: config/locales/es.yml [hunk 3]
```diff
@@ -168,6 +168,11 @@ es:
         unknown: No Se
       language:
         English: Inglés
+      practical_support:
+        companion: Compañera
+        lodging: Alojamiento
```
Comment (montanezp) [inline config/locales/es.yml]: Hospedaje

File: config/locales/es.yml [hunk 4]
```diff
@@ -168,6 +168,11 @@ es:
         unknown: No Se
       language:
         English: Inglés
+      practical_support:
+        companion: Compañera
```
Comment (montanezp) [inline config/locales/es.yml]: Acompañante

File: config/locales/en.yml (hunk 1 above)
Comment (lomky) [reply config/locales/en.yml]: Thanks for gathering the feedback!

**Pull request merged**